product_supplierinfo_for_customer_sale/models/sale_order_line.py

```diff
@@ -11,21 +11,30 @@ class SaleOrderLine(models.Model):
 
     product_customer_code = fields.Char(
         compute="_compute_product_customer_code",
+        store=True,
```

Comment:
This is a no-go. You can't convert a field to stored on a table that can be populated with a lot of records.

Reply:
Thank you for the review, while I get your point I will like to ask how do you would you implement such a feature.

My approach derives by a similar implementation found here https://github.com/OCA/stock-logistics-workflow/blob/14.0/product_supplierinfo_for_customer_picking/models/stock_move.py .
The situation is very similar so I thought that was a possible solution. 
For backward compatibility I can also make a migration script for the `store`  attribute of the field.

But I will like to have you opinion first.

Reply:
One possibility is to not store the field, and implement the `search` attribute

Reply:
@aleuffre Thank you for your suggestion. I have implemented the changes based on your feedback.